sharding/contracts/validator_manager.sol

```diff
@@ -0,0 +1,356 @@
+pragma solidity ^0.4.19;
+
+import "RLP.sol";
+
+interface ValidatorContract {
+  function withdraw(bytes32 _sig) returns(bool);
+}
+
+interface SigHasherContract {
+  // function () returns(bytes32);
+}
+
+contract VMC {
+  using RLP for RLP.RLPItem;
+  using RLP for RLP.Iterator;
+  using RLP for bytes;
+  
+  struct Validator {
+    // Amount of wei the validator holds
+    uint deposit;
+    // The address which the validator's signatures must verify to (to be later replaced with validation code)
+    ValidatorContract validationCodeAddr;
```

Comment:
We should store the validator account. Perhaps having the returnAddr and the validator address be the same

Reply:
Maybe there's no need for `validationCodeAddr` anymore then. We can store`validatorAddr` instead